docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+# Small local network of three nodes using the fastest mean16
+version: '3'
+services:
+  node1:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile-epoch
+    image: aetrnty/epoch
+    hostname: node1
+    environment:
+      EPOCH_CONFIG: /home/epoch/epoch.yaml
+    volumes:
+      - ${PWD}/docker/epoch_node1_mean16.yaml:/home/epoch/epoch.yaml
+      - node1_db:/home/epoch/node/data/mnesia
+      - node1_keys:/home/epoch/node/keys
+
+  node2:
```

Comment:
I made some tests about removing the node2 and node3 explicit configurations here, so there would only be a `service` called `node`. The user of this could then scale to any number of nodes he wants. It worked basically, but it would break the setup with NGINX. Also as (I now know) a usual setup always consists of 3 nodes for developers, so my idea may be useful it later deployments in large scale.